src/components/Modals/ImagePreviewModal/ImagePreviewModal.jsx

```diff
@@ -11,15 +14,55 @@ import {
   Text,
   VStack,
 } from "@chakra-ui/react";
-import React from "react";
-
-import { ChevronLeftIcon, ChevronRightIcon } from "@chakra-ui/icons";
-import { useEffect } from "react";
+import React, { useEffect, useState } from "react";
+import { FaRegThumbsDown, FaRegThumbsUp } from "react-icons/fa";
+import { updateCardById } from "../../../actions/Card";
 
-const ImagePreviewModal = ({ isOpen, onClose, cardImages, cardComments }) => {
+const ImagePreviewModal = ({
+  isOpen,
+  onClose,
+  cardImages,
+  cardComments,
+  cardId,
+  setCards,
+}) => {
   const [currentImageIndex, setCurrentImageIndex] = React.useState(0);
   const [showBackArrow, setShowBackArrow] = React.useState(false);
   const [showNextArrow, setShowNextArrow] = React.useState(false);
+  const [selectedButton, setSelectedButton] = useState(null);
+
+  const handleClick = async (button) => {
+    try {
+      let updatedCard;
+      if (selectedButton === button) {
+        // unselect the currently selected button
+        setSelectedButton(null);
+        // update card with thumbs value set to null
+        updatedCard = await updateCardById(cardId, {
+          thumbsUp: null,
```

Comment:
still a bit confused on why this is null. if this is supposed to be for unlike, make the operation more readable. Rn as a dev, I can't really understand that this is an unlike operation.